languages/en.ini

```diff
@@ -1221,6 +1221,8 @@ renew_success = "Renewal Successful"
 renew_success_summary = "Successfully renewed {count, plural, =1 {1 item} other {# items}}."
 Renewed = "Renewed"
 Request full text = "Request full text"
+request_group_fulfillment_method_delivery = "Delivery to your location"
+request_group_fulfillment_method_hold_shelf = "Pick up at the library"
```

Comment:
I could just shorten the keys to "request_group_hold_shelf" etc. But I think it's valuable to make clear that these keys are specifically from the hard-coded fulfillment methods, unlike other request group keys that may be user-configurable.

As far as the (default) values, I welcome input. It could be as simple as "Pick up" and "Delivery".

Reply:
I don't have strong opinions, but I do wonder if "at the library" is too specific for all cases and being vague might be better. Interested how others feel!

Reply:
I've shortened it to just "Pick up" and "Delivery".  Guessing implementers may want to customize.  "Pick up" could be "Hold shelf" if we think that is a generally understood term.